Rakefile

```diff
@@ -4,8 +4,20 @@ require('rake')
 require('rspec/core/rake_task')
 load('environment.rb')
 
-RSpec::Core::RakeTask.new(:test) do |t|
-  t.pattern = Dir.glob('spec/**/*_test.rb')
+desc "Run all tasks"
+task :test do
+  Rake::Task['test:unit'].execute 
+  Rake::Task['test:integration'].execute    
+end
+
+namespace :test do
+  RSpec::Core::RakeTask.new(:unit) do |t|
+    t.pattern = Dir.glob('spec/**/*_test.rb').reject{|t| t.include?("_integration_test.rb")}
+  end
+  RSpec::Core::RakeTask.new(:integration) do |t|
+    ENV["RUN_INTEGRATION_TESTS"] ||= 'true'
```

Comment:
no longer needed

Reply:
Ah - I thought I caught all the references to that var but my IDE was only searching in .rb files - good catch! Updating now.